client/reader/components/icons/a8c-conversations-icon.jsx

```diff
@@ -0,0 +1,17 @@
+export default function ReaderA8cConversationsIcon() {
+	return (
+		<svg
+			className="sidebar__menu-icon sidebar_svg sidebar_svg-a8c-conversations"
```

Comment:
Are we using the `sidebar_svg` class for anything? Seems like we could omit it. That applies to the rest of the SVG components introduced here.